app/services/user-agent-service/index.js

```diff
@@ -10,18 +10,26 @@
  specific language governing permissions and limitations under the License.
  */
 
-import path from 'path';
 import * as userAgentService from './server';
 import { ProfileStorage } from './storage';
-import * as endpoints from '../../shared/constants/endpoints';
 
 process.on('uncaughtException', console.error);
 process.on('unhandledRejection', console.error);
 
-ProfileStorage.open(path.join(__dirname, '..', '..')).then(async function (profileStorage) {
+export async function UserAgentService(options = {}) {
```

Comment:
`app/main/` has `userAgentService()`, and we have `UserAgentService` here. Can we reduce the collision by renaming one of them?